### PR TITLE
Feature/add example for sdk nft part

### PR DESCRIPTION
**Pull request by MKdir98**
For [issue1](https://github.com/ton-community/gamefi-sdk/issues/1)

My telegram id: [MK_dir98](https://t.me/MK_dir98)

### CODE REVIEW

File: examples/create-nft-collection.ts [hunk 1]
```diff
@@ -0,0 +1,37 @@
+import { Address, toNano } from '@ton/core';
+import { GameFiSDK, createWalletV4 } from '../src/index";
```
Comment (krigga) [inline examples/create-nft-collection.ts:2]: There's a mismatched quote here

File: examples/create-nft-collection.ts [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+        collectionContent: {
+            uri: 'test-uri',
+            name: 'test-name',
+            description: 'lorem ipsum',
+            image: 'test-image',
+        },
+        commonContent: 'test-content'
+    }, {
+        premint: {
+            itemIndex: 1n,
```
Comment (krigga) [inline examples/create-nft-collection.ts:26]: Collection items need to start from index 0

File: examples/create-nft-collection.ts [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+    });
+
+    console.log('Using wallet', sdk.sender?.address);
+
+    const nftCollection = await sdk.createNftCollection({
+        collectionContent: {
+            uri: 'test-uri',
+            name: 'test-name',
+            description: 'lorem ipsum',
+            image: 'test-image',
```
Comment (krigga) [inline examples/create-nft-collection.ts:21]: Since this isn't a valid image, it's better to remove it

File: examples/create-nft-collection.ts [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        },
+        commonContent: 'test-content'
+    }, {
+        premint: {
+            itemIndex: 1n,
+            itemParams: {
+                individualContent: 'test',
```
Comment (krigga) [inline examples/create-nft-collection.ts:28]: This would normally be something like `0.json`

File: examples/create-nft-collection.ts [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+                owner: Address.parse('test-address')
+            }
```
Comment (krigga) [inline examples/create-nft-collection.ts:30]: Please add a comma `,` here

File: examples/create-nft-collection.ts [hunk 6]
```diff
@@ -0,0 +1,37 @@
...
+            description: 'lorem ipsum',
+            image: 'test-image',
+        },
+        commonContent: 'test-content'
+    }, {
+        premint: {
+            itemIndex: 1n,
+            itemParams: {
+                individualContent: 'test',
+                owner: Address.parse('test-address')
```
Comment (krigga) [inline examples/create-nft-collection.ts:29]: I think `sdk.sender!.address!` would be better here

File: examples/mint-nft-items.ts [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+            pinataSecretKey: process.env.PINATA_SECRET!,
+        },
+        api: 'testnet',
+        wallet: await createWalletV4(process.env.MNEMONIC!),
+    });
+
+    console.log('Using wallet', sdk.sender?.address);
+
+    const nftCollection = sdk.openNftCollection(Address.parse('my-nft-collection-address'));
+    nftCollection.sendMint({
```
Comment (krigga) [inline examples/mint-nft-items.ts:17]: This needs to be awaited

File: examples/mint-nft-items.ts [hunk 8]
```diff
@@ -0,0 +1,26 @@
...
+        itemIndex: 2n,
```
Comment (krigga) [inline examples/mint-nft-items.ts:18]: Since we decreased the first item's index, decrease this one too (to 1)

File: examples/mint-nft-items.ts [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+        itemParams: {
```
Comment (krigga) [inline examples/mint-nft-items.ts:19]: Same comments apply to these params as in the `create-nft-collection` example

File: examples/transfer-nft-item.ts [hunk 10]
```diff
@@ -0,0 +1,24 @@
+import { Address, toNano } from '@ton/core";
```
Comment (krigga) [inline examples/transfer-nft-item.ts:1]: Mismatched quotes

File: examples/transfer-nft-item.ts [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+            pinataSecretKey: process.env.PINATA_SECRET!,
+        },
+        api: 'testnet',
+        wallet: await createWalletV4(process.env.MNEMONIC!),
+    });
+
+    console.log('Using wallet', sdk.sender?.address);
+
+    const nftCollection = sdk.openNftCollection(Address.parse('my-nft-collection-address'));
+    const nftItem = await nftCollection.getItem(1n);
```
Comment (krigga) [inline examples/transfer-nft-item.ts:17]: Change this index to 0 too

File: examples/transfer-nft-item.ts [hunk 12]
```diff
@@ -0,0 +1,24 @@
...
+    await nftItem.sendTransfer({
+        to: Address.parse('any-address'),
+        value: toNano('0.03'),
+    })
```
Comment (krigga) [inline examples/transfer-nft-item.ts:21]: Please add a semicolon `;`

**Review by krigga — CHANGES_REQUESTED** [review]

File: examples/transfer-nft-item.ts (hunk 12 above)
Comment (MKdir98) [reply examples/transfer-nft-item.ts:21]: It's ts. It's Ok?

**Pull request closed without merging**